airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/steps/bump_version.py

```diff
@@ -137,6 +137,12 @@ def get_bumped_version(version: str | None, bump_type: str) -> str:
             new_version = current_version.bump_minor()
         elif bump_type == "major":
             new_version = current_version.bump_major()
+        elif bump_type.startswith("version:"):
+            version_str = bump_type.split("version:", 1)[1]
+            if semver.VersionInfo.is_valid(version_str):
+                return version_str
+            else:
+                raise ValueError(f"Invalid version: {version_str}")
```

Comment:
We will have already turned `bump_type` into the explicit version on the input, right? I think this should just be setting it directly, not pulling out version again. 

Though re-checking its an actual version might still be helpful since it helps us catch when someone wants to use `micro` instead of `patch` for example

Reply:
I thought about having the click type transform `version:1.2.3` to `1.2.3` but thought it was best to leave the parsing closer to where its used

Happy to change that decision though

Reply:
Oh no, you're right, i just read it incorrectly!

Reply:
This looks fine to me